### PR TITLE
[WIN32U_VISTA] Add win32u.dll for Vista

### PR DESCRIPTION
**Pull request by tkreuzer**
## Purpose

Add win32u.dll for vista (for testing)

### CODE REVIEW

File: modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt [hunk 1]
```diff
@@ -9,3 +9,8 @@ add_library(win32u_vista MODULE
 
 set_module_type(win32u_vista module)
 add_dependencies(win32u_vista psdk)
+add_rostests_file(TARGET win32u_vista)
+
+if (STACK_PROTECTOR)
```
Comment (katahiromz) [inline modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt:14]: `STACK_PROTECTOR` and `-fno-stack-protector` logically mismatch.

File: modules/rostests/apitests/win32u/CMakeLists.txt [hunk 2]
```diff
@@ -2,4 +2,4 @@
 add_subdirectory(win32u_ros)
 add_subdirectory(win32u_xpsp2)
 add_subdirectory(win32u_2k3sp2)
-#add_subdirectory(win32u_vista)
+add_subdirectory(win32u_vista)
```
Comment (SergeGautherie) [inline modules/rostests/apitests/win32u/CMakeLists.txt:5]: Additional suggestion:
```suggestion
add_subdirectory(win32u_vistasp2)
```

File: modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt (hunk 1 above)
Comment (learn-more) [reply modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt:14]: That is probably to disable stack protector in this library

**Review by learn-more — APPROVED** [review]

**Review by DarkFire01 — APPROVED** [review]

File: modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt (hunk 1 above)
Comment (tkreuzer) [reply modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt:14]: Yes, I copy-pasted it from win32u_ros. @zefklop put it there to "Fix GCC -fstack-protector usage" in rev c8d07514c8e0bc235e180f6990f914ee86880f8c

**Pull request merged**